zebrad/src/components/inbound/cached_peer_addr_response.rs

```diff
@@ -0,0 +1,96 @@
+//! Periodically-refreshed GetAddr response for the inbound service.
+//!
+//! Used to avoid giving out Zebra's entire address book over a short duration.
+
+use std::{
+    sync::{Mutex, TryLockError},
+    time::Instant,
+};
+
+use super::*;
+
+/// The maximum duration that a `CachedPeerAddrResponse` is considered fresh before the inbound service
+/// should get new peer addresses from the address book to send as a `GetAddr` response.
+pub const INBOUND_CACHED_ADDRS_REFRESH_INTERVAL: Duration = Duration::from_secs(10 * 60);
```

Comment:
Optional Nitpick: I think this constant name could be confusing, because it's used in the inbound service, but the address book contains peer addresses for outbound connections.

Similarly for the next constant.

```rust
pub const CACHED_ADDRS_REFRESH_INTERVAL: Duration = Duration::from_secs(10 * 60);
```

Reply:
Updated in https://github.com/ZcashFoundation/zebra/pull/7955/commits/1e3fb0eb7d389c697586d07bd5394ee1f9dee5db.